src/components/Header.js

```diff
@@ -42,11 +42,10 @@ class Header extends React.Component {
   }
 
   // Filters what gets displayed based on the user inputs
-  /*
-  fetchCurRecipes = async () => {
-    await fetch(`/recipes?page=${this.state.currentPage}&limit=${this.state.pageSize}&` +
-     `${this.createIngredientQueryParam()}name=${this.state.name}&min=${this.state.minTime}` +
-     `&max=${this.state.maxTime}&username=${this.state.createdBy}&recipes=${LZString.decompress(localStorage.getItem("recipes"))}`, {
+  fetchRecipes = () => {
+    fetch(`/recipes?page=${this.state.currentPage}&limit=${this.state.pageSize}&` +
+     `${this.createIngredientQueryParam()}name=${this.state.name}&minTime=${this.state.minTime}` +
```

Comment:
if we're keeping with 80 character limit, these lines are a bit too long. Also the indentation is weird

Reply:
fixed length of line and indentation